todaythediary/src/components/todo/Form.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { BsPlusCircleFill } from 'react-icons/bs';
+
+const Form = ({inputText,setInputText, todos, setTodos}) =>{
+
+    const inputTextHandler = (e) =>{
+        setInputText(e.target.value);
+    };
+    const submitTodoHandler =(e) =>{
+        e.preventDefault();
+        setTodos([...todos, {text:inputText , completed: false, id: Math.random()*10000}])
```

Comment:
Can you explain why `Math.random()*10000` is used here for the ID instead of ordered numbers? Thanks!

Reply:
i have tried to implement unique id using uuid but failed :( . No problem. I will try again and update soon as possible .